blocks/components/editable/index.js

```diff
@@ -127,6 +146,22 @@ export default class Editable extends wp.element.Component {
 		}
 	}
 
+	componentWillReceiveProps( nextProps ) {
+		forEach( nextProps.formats, ( state, format ) => {
+			const currentState = this.formats[ format ] || false;
```

Comment:
Yes, It can be different than `this.props.formats` right after we trigger the `onFormatChange` callback.